.github/workflows/ember-compare-asset-sizes.yml

```diff
@@ -0,0 +1,32 @@
+name: Ember compare asset sizes
+
+on:
+  workflow_call:
+    secrets:
+      npm-token:
+        required: true
+      ssh-private-key:
+        required: true
+
+permissions:
+  contents: read
+  pull-requests: write
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Setup npm token for private packages
+      run: echo //registry.npmjs.org/:_authToken=${{ secrets.npm-token }} > ~/.npmrc
+    - name: Setup SSH for Github Auth
+      uses: webfactory/ssh-agent@v0.5.4
+      with:
+        ssh-private-key: ${{ secrets.ssh-private-key }}
+    - uses: actions/checkout@v2.3.4
+      with:
+        # when using pull_request_target, need to specify what sha to compare against (otherwise it does PR base)
+        ref: ${{ github.event.pull_request.head.sha }}
+        fetch-depth: 0
+    - uses: simplabs/ember-asset-size-action@v2
+      with:
+        repo-token: "${{ secrets.GITHUB_TOKEN }}"
```

Comment:
`GITHUB_TOKEN` is always provided by caller workflow, no need to have it specified in the secrets list at the top